### ISSUE TITLE
Swagger does not show endpoint

### ISSUE THREAD

**Issue by Echarnus** [GFlisch/Arc4u.Guidance.Doc#188]
**Information**
If the bug is linked to the Arc4u framework, please go to the [Arc4u repo](https://github.com/GFlisch/Arc4u/issues).

**Describe the bug**

When generating a new controller to the facade project it is not shown in the swagger.json 

**To Reproduce**

1. Right click at the facade project
2. Add REST api
3. Select entity you wish to generate the controller for

**Expected behavior**

The controller and its entities should be defined within the swagger.json

**Screenshots**

**Desktop (please complete the following information):**
 - Visual Studio Version [e.g. 17.4.11]
17.5.4
 - Guidance Version [e.g. 2022.2.1] (Go to Visual Studio extensions to check the version)
2022.2.1.12
**Additional context**

**Comment by GFlisch** [comment]
Hi @Echarnus, can you give me more information.

What do you mean by "The controller and its entities should be defined within the swagger.json". Did you add a controller and the swagger page was not showing the new api?

**Comment by Echarnus** [comment]
Yes indeed. Upon adding the controller based on the entity I noticed in this case it didn't generate the actions based on the generated business layer. All though, I have to be honest. Later on it did generate them. Don't really know what changed though..

**Comment by rdarko** [comment]
Can you please re-test this? If its working we would like to close the ticket.
